src/alert/alert.tsx

```diff
@@ -30,11 +30,11 @@ function Alert({ id, children, tone, closeLabel, onClose }: AlertProps) {
             id={id}
             role="alert"
             aria-live="polite"
-            borderRadius="standard"
+            borderRadius="full"
             className={[styles.container, getClassNames(styles, 'tone', tone)]}
         >
-            <Columns space="small" alignY="top">
-                <Column width="content">
+            <Columns space="small" alignY="center">
+                <Column width="content" exceptionallySetClassName={styles.icon}>
```

Comment:
Setting the class exceptionally preferable to apply styles directly to the `svg`?

Reply:
We can also apply it directly to the SVG 👍 updated

Reply:
haha, my curiosity was honest and totally not a hint towards a change. 😅

I feel kinda _"meh"_ when looking at this differences and wanted to take the opportunity to learn.